components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/rust/LibFxAFFI.kt

```diff
@@ -85,7 +85,14 @@ internal interface LibFxAFFI : Library {
     )
     fun fxa_send_tab(fxa: FxaHandle, targetDeviceId: String, title: String, url: String, e: RustError.ByReference)
 
-    fun fxa_migrate_from_session_token(fxa: FxaHandle, sessionToken: String, kSync: String, kXCS: String, e: RustError.ByReference)
+    fun fxa_migrate_from_session_token(
+        fxa: FxaHandle,
+        sessionToken: String,
+        kSync: String,
+        kXCS: String,
+        copySessionToken: Boolean,
```

Comment:
I'd lightly prefer to have separate `fxa_migrate_from_session_token` and `fxa_copy_from_session_token` functions rather than a boolean flag here, since consistency would make it easier to follow the implementation down through the FFI layer and into the rust code. But I'm not going to insist on it :-)